github-issue/in.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import requests
+import json
+import sys
+import os
+
+def _in(instream):
+    payload = json.load(instream)
+    source = payload["source"]
+
+    token = source["token"]
+    repository = source["repository"]
+    endpoint = "https://api.github.com/repos/" + repository + "/issues/" + number
+    headers = {'Authorization': 'token ' + token}
+
+    connection = requests.get(endpoint, headers=headers)
+    i = json.loads(connection.text)
+
+    for item in issueNumbers:
+        number = item
+        endpoint = "https://api.github.com/repos/" + repository + "/issues/" + number
+        connection = requests.get(endpoint, headers=headers)
+        i = json.loads(connection.text)
+        if issueNumber[item] == i['updated_at']:
+            if not os.path.exists(str(number)):
+                os.makedirs(str(number))
```

Comment:
this pattern of pulling down all the info for multiple issues and putting them in separate directories is not necessary for this resource i don't think, we always just want to pull in the one thing